Sofa/framework/Testing/CMakeLists.txt

```diff
@@ -39,6 +83,8 @@ sofa_find_package(Sofa.Simulation.Graph REQUIRED)
 
 add_library(${PROJECT_NAME} SHARED ${HEADER_FILES} ${SOURCE_FILES})
 target_link_libraries(${PROJECT_NAME} PUBLIC Sofa.Helper Sofa.DefaultType Sofa.Core Sofa.Simulation.Graph gtest SofaGTestMain )
```

Comment:
Use `GTest::gtest` target instead of `gtest`. This is the one defined by the cmake file for external package and which is also handled by the `sofa_create_package_with_targets` for the case of using fetched code.

Reply:
You must also update the same target that is used in `SofaGTestMain/CMakeLists.txt` for `SofaGTestMain